modules/subnets/main.tf

```diff
@@ -31,6 +31,8 @@ resource "google_compute_subnetwork" "subnetwork" {
   ip_cidr_range            = each.value.subnet_ip
   region                   = each.value.subnet_region
   private_ip_google_access = lookup(each.value, "subnet_private_access", "false")
+  purpose                  = lookup(each.value, "subnet_purpose", null)
```

Comment:
followed the `subnet_xxx` pattern here, but `subnet-beta` declares these as just `role` and `purpose` (no prefix)

Reply:
We can go with subnet prefix since others follow this pattern in this module